test/functions/ferguson/ref-pair/sparse-array-query-irv-locale.chpl

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
I'm not see the value in this test. Since there's no distribution, shouldn't it always print out zero? Doesn't this work on master already? Would a sparse Block array be better?